openwb.conf

```diff
@@ -262,3 +262,11 @@ wizzarddone=0
 datenschutzack=0
 virtual_ip_eth0='192.168.193.5'
 virtual_ip_wlan0='192.168.193.6'
+aiwaysuser=''
```

Comment:
Damit bestehende Installationen beim Update die Einstellungen erhalten können, müssen diese in der `runs/updateConfig.sh` ergänzt werden.